fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -0,0 +1,2 @@
+Free (as in freedom) Android client for Pixelfed, the federated image sharing platform.
```

Comment:
This is too long. Max length is 80 chars, after that it will be truncated. As "free" is implied by the GPL, and "Android" is pretty clear when published to a repo of Android apps, may I suggest:

    client for Pixelfed, the federated image sharing platform

The trailing newline will probably be truncated automatically.